app/views/search/index.html.erb

```diff
@@ -1,15 +1,46 @@
-<h1>Search Results</h1>
+<div class="l-constrained">
+  <section class="l-results__main">
+    <%= heading_tag( t('search.page_title'), level: 1) %>
 
-<div>
-  Showing <span class="t-first-record"><%= @result.first_record %></span> to <span class="t-last-record"><%= @result.last_record %></span> of <span class="t-total-records"><%= @result.total_records %></span>
-</div>
+    <p><%= t('search.summary.showing') %> <span class="l-results__emphasised t-first-record"><%= @result.first_record %></span> <%= t('search.summary.to') %> <span class="l-results__emphasised t-last-record"><%= @result.last_record %></span> <%= t('search.summary.of') %> <span class=" l-results__emphasised t-total-records"><%= @result.total_records %></span> <%= t('search.summary.firms') %></p>
+
+    <ol>
+      <% @result.firms.each do |firm| %>
+        <li class="t-firm">
+          <span class="t-name"><%= firm.name %></span>, advisers: <%= firm.total_advisers %> (<%= firm.closest_adviser %>)
+        </li>
+      <% end %>
+    </ol>
+
+    <%= paginate @result %>
+  </section>
+
+  <div class="l-results__side">
+    <section class="l-results__filter">
+    </section>
 
-<ol>
-  <% @result.firms.each do |firm| %>
-    <li class="t-firm">
-      <span class="t-name"><%= firm.name %></span>, advisers: <%= firm.total_advisers %> (<%= firm.closest_adviser %>)
-    </li>
-  <% end %>
-</ol>
+    <section class="education">
```

Comment:
This section and the one below would be perfect candidates for extraction to a partial.